feature-group-definitions/view-transitions.yml

```
@@ -1,4 +1,5 @@
 name: View transitions
+description: The View Transitions API transitions between different states of a document, or between different documents.
```

I think we'll need another feature for MPA view transitions, which is the "or between different documents" bit here:

```suggestion
description: View transitions allow you to create animated visual transitions between different states of a document.
```

---

Actually, ignore this suggestion. The SPA shipped first in Chrome:
https://chromiumdash.appspot.com/commit/c03f83e180726ba898a2191f616aa7df9c581510

But perhaps we will treat that as a partial implementation later, rather than having two features.

Leaving only the good parts of my suggestion:

```suggestion
description: View transitions allow you to create animated visual transitions between different states of a document, or between different documents.
```